peregrine/analysis/tracking_loop.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python
```

Comment:
See above comments on CSV format. This analysis tool looks good. Maybe we could standardize a set of input arguments and use something similar in `run.py` to what you have here, and remove `initSettings.py` all together. What do you think?